wikimedia/wikipedia-ios
# Allow iPads to pop into images/articles

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T258109

### Notes
* The animator can have 2 commit styles - `dismiss` or `pop`. The safety check being removed in this PR was added in the [first commit](https://github.com/wikimedia/wikipedia-ios/commit/921c96ed950c15c811c6adedb0d96a0ff007d9ad#diff-b3a4e8778a7c51d6747010167b742d27ffe6d9bc1613fb09070e7a0340f2c9e0) while working on peek-and-pop and PCS, and I can't find any reason for it to exist beyond defensive coding. I believe in all cases we want to allow peeked article elements to pop, so it is safe to remove. (And we don't currently have any other method that accesses the animator. IMO: It's not worth adding another delegate method, but if we already had one that accessed the animator we could have updated the commit style there.)
* I'm guessing the problem doesn't exist on iPhone because the default commit style is likely `pop` on phones.
* The issue is about images, but article previews also don't currently pop. This fixes the problem.

### Test Steps
1. Run on an iPad. Within an article, ensure images and articles can be peeked, then popped.
